src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,203 @@
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+import javafx.util.StringConverter;
+
+import com.esri.arcgisruntime.arcgisservices.RenderingRuleInfo;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.ImageServiceRaster;
+import com.esri.arcgisruntime.raster.RenderingRule;
+
+public class RasterRenderingRuleSample extends Application {
+
+  private ComboBox<RenderingRuleInfo> renderingRuleInfoComboBox;
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      final String IMAGE_SERVICE_RASTER_URI = "http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer";
+
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Raster Rendering Rule Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a control panel
+      VBox controlsVBox = new VBox();
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.6)"), CornerRadii.EMPTY, Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setSpacing(8);
+      controlsVBox.setMaxSize(260, 160);
+      controlsVBox.getStyleClass().add("panel-region");
+
+      // create progress indicator
+      ProgressIndicator progressIndicator = new ProgressIndicator();
+      progressIndicator.setVisible(false);
+
+      // create drop down menu of Rendering Rules
+      renderingRuleInfoComboBox = new ComboBox<>();
+      renderingRuleInfoComboBox.setMaxWidth(260.0);
+      renderingRuleInfoComboBox.setConverter(new StringConverter<>() {
+        @Override
+        public String toString(RenderingRuleInfo renderingRuleInfo) {
+
+          return renderingRuleInfo != null ? renderingRuleInfo.getName() : "";
+        }
+
+        @Override
+        public RenderingRuleInfo fromString(String string) {
+          return null;
+        }
+      });
+
+      // create a label for the Rendering Rule info
+      Label renderingRuleInfoLabel = new Label("");
+      renderingRuleInfoLabel.setMaxWidth(220);
```

Comment:
Don't think this is needed. Its width should be constrained by the VBox it's inside.

Reply:
It is not! I think I ended up having both because I tried `.setMaxWidth()` assuming it'll cause the text to wrap. Good to know.